sqllogictest-bin/src/lib.rs

```diff
@@ -170,8 +170,17 @@ pub async fn main_okk() -> Result<()> {
         Color::Auto => {}
     }
 
-    let files = glob::glob(&files).context("failed to read glob pattern")?;
-    let files = files.into_iter().try_collect::<_, Vec<_>, _>()?;
+    let files: Vec<PathBuf> = files
+        .into_iter()
+        .map(|filename| glob::glob(&filename).context("failed to read glob pattern"))
+        .try_collect::<_, Vec<_>, _>()?
+        .into_iter()
+        .map(|glob| glob.try_collect::<_, Vec<_>, _>())
+        .try_collect::<_, Vec<_>, _>()?
+        .into_iter()
+        .flatten()
+        .collect();
+
```

Comment:
The repeated `try_collect` and `into_iter` don't look very nice... Maybe simple for loops are just more ergonomic in this case.

Reply:
Alternatively you can use `and_then` in the first `map`. In this way you can save on `try_collect`

Reply:
Actually I feel for loop might be more expressive…

Reply:
I got brainwashed by the pipe style previously, changed it to `for` and it looks much better.
Much appreciated for the feedback!